aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfiguration.java

```diff
@@ -206,4 +288,41 @@ public DataStoreConfiguration build() {
             );
         }
     }
+
+    enum ConfigKey {
+        /**
+         * At most one base sync will be performed within this interval of time.
+         * The interval is expressed in milliseconds.
+         */
+        SYNC_INTERVAL("syncInterval"),
+        SYNC_PAGE_SIZE("syncPageSize"),
```

Comment:
Should we write descriptions for these as well?

Reply:
That was my miss. I'll add that. Checkstyle didn't flag because the interface is not public, but it definitely warrants comments.